src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -0,0 +1,18 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Responds with a hard-coded message for testing purposes. */
```

Comment:
Move this comment right above the response.getOutputStream. We want to make sure that the comment is as close to the hard-coded message as possible. Also when commenting inside a function, you will need to change your comment styling from /** */ to //.